src/store/actions/appActions.js

```diff
@@ -0,0 +1,8 @@
+import { ActionType } from './actionType';
+
+export const ActionCreator = {
+	reset: () => ({
```

Comment:
based on the improvements above you should change function reset: (dispatch) => dispatch({type: ActionType.RESET})